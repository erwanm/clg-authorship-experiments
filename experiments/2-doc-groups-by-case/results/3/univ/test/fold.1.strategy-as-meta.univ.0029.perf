0.394993	0.637600	0.619500
