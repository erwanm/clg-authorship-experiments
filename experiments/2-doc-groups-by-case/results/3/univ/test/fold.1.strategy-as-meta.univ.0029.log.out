apply-config.sh: applying config '/tmp/expe2/3/process/selected-meta-configs/fold.1.strategy-as-meta.univ.0029' to cases './results/3/univ/test/only-cases.tmp.fiUMSlxm'; results written to './results/3/univ/test/fold.1.strategy-as-meta.univ.0029'
apply-config.sh: done.
